sigal/gallery.py

```diff
@@ -371,7 +379,7 @@ def thumbnail(self):
                     except:
                         self.logger.error("Failed to open %s", f.src_path)
                     else:
-                        if im.size[0] > im.size[1]:
+                        if im.size[0] > im.size[1]: 
```

Comment:
In this block of code there is a call to `PILImage.open` which could be simplified by using the new property, if you want to change that.